modules/sdk/project-templates/project-templates-social-bookmark/build.gradle

```diff
@@ -1,4 +1,15 @@
 dependencies {
-	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.1.6"
+	compile project(":sdk:project-templates:project-templates")
+
 	compileOnly group: "org.apache.maven.archetype", name: "archetype-common", version: "2.4"
+}
+
+generateSampleProject {
+	String sampleProjectName = "foo"
```

Comment:
Please remove `sampleProjectName` and hardcode it to reduce the number of lines in build.gradle